service/src/main/resources/HTMLTemplate.html

```diff
@@ -525,24 +525,28 @@ <h3 class="heading heading--two">Child
         case_data.children.firstChild is not empty and
         case_data.children.firstChild.livingSituation is not empty %}
           <span class="u-block">{{ case_data.children.firstChild.livingSituation }}</span>
-          {% if case_data.children.firstChild.address is not empty %}
-            {% if case_data.children.firstChild.address.AddressLine1 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine1 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.AddressLine2 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine2 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.AddressLine3 is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.AddressLine3 }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.PostTown is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.PostTown }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.PostCode is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.PostCode }}</span>
-            {% endif %}
-            {% if case_data.children.firstChild.address.Country is not empty %}
-              <span class="u-comma-before">{{ case_data.children.firstChild.address.Country }}</span>
+          {% if case_data.children.firstChild.detailsHidden is not empty and case_data.children.firstChild.detailsHidden == "Yes" %}
+            Confidential
+          {% else %}
```

Comment:
Is it worth changes these to elseif blocks? Perhaps the condition might be too lengthy to merit this change. Thoughts?

Reply:
I thought if and then an else would be less code. Essentially I want to show address when detailsHidden is empty or when detailsHidden == No. If I did ifelse then I would have address code twice, no?

Reply:
You would need to do else if detailsHidden is not empty || detailsHidden == "No" and address is not empty...

Bit too lengthy, inner if statement is cleaner.